en/docs/install-and-setup/setup/api-controller/managing-apis-in-kubernetes/resources/kustomization.yaml

```diff
@@ -0,0 +1,46 @@
+namePrefix: production-
+
+commonLabels:
+  app: my-hello
+  kustomize-used: dtrue
+
+commonAnnotations:
+  note: Hello, I am production!
+
+resources:
+- api_crd.yaml
+
+configurations:
+- kustomize-config.yaml
+
+configMapGenerator:
+# - name: deployment-configs
+- name: PizzaShackAPI-params
+- name: PizzaShackAPI-cm
+  files:
+  - PizzaShackAPI_1.0.0.zip
+
+secretGenerator:
+- name: PizzaShackAPI-certs
+  files:
+    - "certificates/tls.cert"
+    - "certificates/tls.key"
+  type: "kubernetes.io/tls"
+
+patchesStrategicMerge:
+# - deployment-configs.yaml
+ - PizzaShackAPI-params.yaml
+
+# kustomized-used: true
+
+#kubect label change kustomized-used: false
+
+#kubectl kustomize over/ | kubectl apply -f 
+
+#kubectl delete cm -l kustomized-used: false
+
```

Comment:
Shall we remove the unwanted newlines here?

Reply:
Fixed with 21f466ce5d99225a24a45351eae0997e87f8c2f5